### DISCUSSION TITLE
Subscriber model

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#1034] [Ideas]
Hello

I have a suggestion to integrate with services like Patreon, so that you can make a subscriber model for comments.

The bottom line is that remark42 is most needed by sites like hugo. On them it is problematic to make paid posts "only for their own". If remark42 had integration with patreon, this would make it possible to somehow monetize projects on engines like hugo.

Also, the minimum subscription, for example $ 1 per month, is a great barrier that clears comments from trolls.

I hope I was able to get my point across

**Comment by umputun** [comment]
If I got it right, you are suggesting adding some way to tie login with a payment system? I.e. allow logins to remark42 for paid users only. Such functionality probably can be implemented as some sort of patrion auth provider as a part of go-pkgz/auth used by remark42 to handle all auth aspects. I have no clue how hard/easy to do it (not familiar with patreon's apis), but feel free to open a ticket in  [go-pkgz/auth](https://github.com/go-pkgz/auth) project, and maybe some contributor will make it happen.

**Reply by uPagge** [reply]
Yes that's right. Of course patreon is not the best system because of their commissions, but alas, I don’t know better.

**Reply by uPagge** [reply]
Created, I hope someone will take it into implementation

https://github.com/go-pkgz/auth/issues/89
